lib/linguist/languages.yml

```diff
@@ -3065,6 +3065,14 @@ MoonScript:
   - moon
   ace_mode: text
   language_id: 238
+Motorola 68K Assembly:
+  type: programming
+  group: Assembly
+  extensions:
+  - ".X68"
+  tm_scope: source.m68k
+  ace_mode: assembly_x86
+  language_id: 477582706
```

Comment:
I note you've not picked a colour for this language. That's not a problem as we'll fall back to the default of `#cccccc` but I wanted to check if this is intentional.

Reply:
It's part of a group so it won't get a color anyway, no?

Reply:
Doh!! Of course. I completely missed that. Thanks @pchaigno